### PR TITLE
Update installation-creating-azure-bootstrap.adoc

### PR DESCRIPTION
**Pull request by nikhilprajapati-world**
As per doc bug raised https://issues.redhat.com/browse/OCPBUGS-29724, I propose to mention in the docs about the parameters to set the  VM size when using the templates,

$ az deployment group create -g ${RESOURCE_GROUP} \
  --template-file "<installation_directory>/04_bootstrap.json" \
  --parameters bootstrapIgnition="${BOOTSTRAP_IGNITION}" \ 
  --parameters baseName="${INFRA_ID}" \
  --parameters bootstramVMSize="Standard_D4s_v3"

Parameters to set the VM size when using the templates which using architecture as ARM/ARCH64

$ az deployment group create -g ${RESOURCE_GROUP} \
  --template-file "<installation_directory>/04_bootstrap.json" \
  --parameters bootstrapIgnition="${BOOTSTRAP_IGNITION}" \ 
  --parameters baseName="${INFRA_ID}" \
  --parameters bootstramVMSize="Standard_D4ps_v5"

Version(s):

Issue:

Link to docs preview:

https://docs.openshift.com/container-platform/4.15/installing/installing_azure/installing-azure-user-infra.html#installation-creating-azure-bootstrap_installing-azure-user-infra

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-creating-azure-bootstrap.adoc [hunk 1]
```diff
@@ -12,12 +11,8 @@ ifeval::["{context}" == "installing-azure-stack-hub-user-infra"]
 :ash:
 :cp: Azure Stack Hub
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-azure-user-provisioned"]
-:azure:
-:cp: Azure
-endif::[]
```
Comment (mjpytlak) [inline modules/installation-creating-azure-bootstrap.adoc:18]: This module is still used by `installing-restricted-networks-azure-user-provisioned`. Did you remove this by mistake?

File: modules/installation-creating-azure-bootstrap.adoc [hunk 2]
```diff
@@ -12,12 +11,8 @@ ifeval::["{context}" == "installing-azure-stack-hub-user-infra"]
...
 
-:_mod-docs-content-type: PROCEDURE
+:_content-type: PROCEDURE
```
Comment (mjpytlak) [inline modules/installation-creating-azure-bootstrap.adoc:15]: The meta data should not be updated. `_mod-docs-content-type: PROCEDURE` is the correct meta data.

File: modules/installation-creating-azure-bootstrap.adoc [hunk 3]
```diff
@@ -96,10 +91,12 @@ ifdef::azure[]
 $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/04_bootstrap.json" \
   --parameters bootstrapIgnition="${BOOTSTRAP_IGNITION}" \ <1>
```
Comment (mjpytlak) [inline modules/installation-creating-azure-bootstrap.adoc:93]:
```suggestion
  --parameters bootstrapIgnition="${BOOTSTRAP_IGNITION}" \// <1>
```

File: modules/installation-creating-azure-bootstrap.adoc [hunk 4]
```diff
@@ -96,10 +91,12 @@ ifdef::azure[]
...
-  --parameters baseName="${INFRA_ID}" <2>
+  --parameters baseName="${INFRA_ID}" \ <2>
```
Comment (mjpytlak) [inline modules/installation-creating-azure-bootstrap.adoc:94]:
```suggestion
  --parameters baseName="${INFRA_ID}" \// <2> 
```

File: modules/installation-creating-azure-bootstrap.adoc [hunk 5]
```diff
@@ -96,10 +91,12 @@ ifdef::azure[]
 $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/04_bootstrap.json" \
   --parameters bootstrapIgnition="${BOOTSTRAP_IGNITION}" \ <1>
-  --parameters baseName="${INFRA_ID}" <2>
+  --parameters baseName="${INFRA_ID}" \ <2> 
+  --parameters bootstramVMSize="${BOOTSTRAP_VM_SIZE}" <3>
 ----
 <1> The bootstrap Ignition content for the bootstrap cluster.
 <2> The base name to be used in resource names; this is usually the cluster's infrastructure ID.
+<3> If the architecture is x86_64 then vm type should be "Standard_D4s_v3", or if the architecture is ARM/ARCH64 then vm  type must be "Standard_D4ps_v5"
```
Comment (mjpytlak) [inline modules/installation-creating-azure-bootstrap.adoc:99]:
```suggestion
<3> If the architecture is x86_64, then vm type should be `Standard_D4s_v3`. If the architecture is ARM/ARCH64, then vm  type must be `Standard_D4ps_v5`.
```

**Review by mjpytlak — COMMENTED** [review]
@nikhilprajapati-world Some feedback based on a peer review.

**Comment by mjpytlak** [comment]
@jinyunma PTAL for QE. Thank you.

**Comment by mjpytlak** [comment]
@kelbrown20 I see that you are assigned to https://issues.redhat.com/browse/OCPBUGS-29724. FYI.

**Comment by kelbrown20** [comment]
@mjpytlak @nikhilprajapati-world This fix has already been made in https://github.com/openshift/openshift-docs/pull/72310#pullrequestreview-1907094615

**Comment by mjpytlak** [comment]
@kelbrown20 Well I am glad I reached out. :smile: 

Are we good to close this PR out, as well as https://github.com/openshift/openshift-docs/pull/72371#issuecomment-1979005607 and https://github.com/openshift/openshift-docs/pull/72374#issuecomment-1979007328

**Comment by kelbrown20** [comment]
I think it would be good to close out

**Comment by mjpytlak** [comment]
Closing at @kelbrown20 request.

**Pull request closed without merging**